### PR TITLE
Update economy.js

### PR DESCRIPTION
**Pull request by FlameOverload**

### CODE REVIEW

File: chat-plugins/economy.js [hunk 1]
```diff
@@ -122,6 +122,7 @@ exports.commands = {
 
 		Economy.writeMoney(targetUser, amount);
 		this.sendReply(Tools.escapeHTML(targetUser) + " has received " + amount + ((amount === 1) ? " buck." : " bucks."));
+		if (Users.getExact(targetUser) && Users.getExact(targetUser)) Users.getExact(targetUser).popup("|modal||html|You have been given " + amount + ((amount === 1) ? " buck" : " bucks") + " by <b><font color='" + Wisp.hashColor(user.name) + "'>" + user.name + "</font></b>" )
```
Comment (SilverTactic) [inline chat-plugins/economy.js]: It should be <code>Users.getExact(targetUser) && Users.getExact(targetUser).connected</code>

File: chat-plugins/economy.js [hunk 2]
```diff
@@ -143,6 +144,7 @@ exports.commands = {
 
 		Economy.writeMoney(targetUser, -amount);
 		this.sendReply("You removed " + amount + ((amount === 1) ? " buck " : " bucks ") + " from " + Tools.escapeHTML(targetUser));
+		if (Users.getExact(targetUser) && Users.getExact(targetUser)) Users.getExact(targetUser).popup("|modal||html|Your " + amount + ((amount === 1) ? " buck" : " bucks") + " has been taken by <b><font color='" + Wisp.hashColor(user.name) + "'>" + user.name + "</font></b>")
```
Comment (SilverTactic) [inline chat-plugins/economy.js]: Same here

File: chat-plugins/economy.js [hunk 3]
```diff
@@ -167,7 +169,7 @@ exports.commands = {
 				Economy.writeMoney(targetUser, amount, () => {
 					this.sendReply("You've sent " + amount + ((amount === 1) ? " buck " : " bucks ") + " to " + targetUser);
 					Economy.logTransaction(user.name + " has transfered " + amount + ((amount === 1) ? " buck " : " bucks ") + " to " + targetUser);
-					if (Users.getExact(targetUser) && Users.getExact(targetUser)) Users.getExact(targetUser).popup(user.name + " has sent you " + amount + ((amount === 1) ? " buck." : " bucks."));
+					if (Users.getExact(targetUser) && Users.getExact(targetUser)) Users.getExact(targetUser).popup("|modal||html|<b><font color='" + Wisp.hashColor(user.name) + "'>" + user.name + "</font></b> has sent you " + amount + ((amount === 1) ? " buck." : " bucks."));
```
Comment (SilverTactic) [inline chat-plugins/economy.js]: Here too

**Comment by panpawn** [comment]
"This branch has conflicts that must be resolved"

**Pull request closed without merging**